GSA/openacr-editor
# Cypress tests

Closes https://github.com/GSA/openacr/issues/208

QA:
1. Should see Cypress test GitHub action run and complete with no errors. Latest run https://github.com/GSA/openacr-editor/runs/4668991229?check_suite_focus=true
2. Should see test results as a comment on this pull request https://github.com/GSA/openacr-editor/pull/23#issuecomment-1003130539.
